flow/project.py

```diff
@@ -687,6 +688,12 @@ def __call__(self, func=None, /, *, directives=None):
         func
             The decorated function.
         """
+        if directives is not None:
+            warnings.warn(
+                "The current directives are deprecated as of flow 0.27. When updating flow "
+                "please look at the documentation for the new style.",
```

Comment:
It would be nice to have a link to where this documentation will be or at least keep it updated as this changes